src/data/faq.json

```diff
@@ -51,10 +51,10 @@
+                            "With version 2.2, the Corona-Warn-App (CWA) offers a new function for detailed technical problem analysis. The error report is intended to help the developers to faster perform a root cause analysis and to eliminate errors more efficiently.<br><b> Important:</b> Please report an issue <b>first</b> to the technical hotline (<a href='#international_phone_numbers' target='_blank' rel='noopener noreferrer'>click here to see the phone numbers</a>) to verify if there is already a known solution. An error report is only required if the CWA team asks for this report.",
+                            "<ol><li>If so, please open an issue on GitHub (<a href='https://github.com/corona-warn-app/cwa-app-android/issues/new/choose' target='_blank' rel='noopener noreferrer'>Android</a> or <a href='https://github.com/corona-warn-app/cwa-app-ios/issues/new/choose' target='_blank' rel='noopener noreferrer'>iOS</a>) and describe the problematic behavior as detailed as possible.</li><li>Start logging in the CWA (<a href='../blog/2021-05-12-corona-warn-app-version-2-2/' target='_blank' rel='noopener noreferrer'>see blog article</a>).<br>- Try to reproduce the faulty behavior or try to generate the error message.<br>- If this attempt was successful, send the error report to the CWA server. The data will be anonymized and encrypted. (If you are interested, you may also save the report locally on your Smartphone.)<br>- In return you will receive a unique ID, which you can also find in the list of all IDs.<br>- Stop the error report.</li><li>Update the GitHub issue with your unique identifier from the server so that the report can be linked to your issue. The GitHub community thereby does <b>not</b> get access to your error report, the report can only be seen and analyzed by the CWA team.</li></ol>",
                             "Remarks:<ul><li>Error reports are automatically deleted after 14 days.</li><li>Error reports that are transmitted to the server without request by the CWA team cannot be analyzed and processed.</li><li>There is no back channel into the Corona-Warn-App for the error reports. The CWA technical team can only communicate via GitHub to an issue.</li></ul>",

```

Comment:
> There is no back channel into the Corona-Warn-App for the error reports. The CWA technical team can only communicate via GitHub to an issue.

This is probably confusing to understand.

better: Feeback about your error reports cannot be received directly through the CWA. Instead, feedback will be communicated through the channel where you reported the issue, i.e Github or your app store review.

Reply:
I agree that this sentence is hard to understand, bu, do we really want to mention the App Store here? 
It was not mentioned in the entry before and I think the preferred way is via GitHub, isn't it?

Reply:
Please see https://github.com/corona-warn-app/cwa-website/pull/1267#issuecomment-841499385